File: DofusApi/Models/ForumReply.cs
using System;

namespace DofusApi.Models
{
    public class ForumReply
    {
        public int ID { get; set; }
        public string Content { get; set; }
        public int ForumCommentID { get; set; }
        public ForumComment ForumComment { get; set; }
        public DateTime CreatedAt { get; set; }

        public int? UserID { get; set; }
        public User User { get; set; }
    }
}

File: DofusSpa/src/app/_models/forum-reply.ts
import { ForumComment } from "./forum-comment";
import { User } from "./user";

export interface ForumReply{
    id : number,
    content : string,
    forumCommentID : number,
    forumComment : ForumComment,
    createdAt : Date,
    userID : number, 
    user : User
}